nexus/db-queries/src/db/datastore/db_metadata.rs

```diff
@@ -830,39 +829,35 @@ impl DataStore {
     pub async fn database_nexus_access_create(
         &self,
         opctx: &OpContext,
-        blueprint: &nexus_types::deployment::Blueprint,
+        blueprint_id: BlueprintUuid,
+        active: &BTreeSet<OmicronZoneUuid>,
```

Comment:
More a note for myself: this is identical to #8936, but using BTreeSets instead of Vec (totally reasonable)